src/dotnet/CarbonAware.WebApi/api.yaml

```diff
@@ -157,4 +240,42 @@ components:
         "marginalCarbonIntensityValue": 750.0,
         "embodiedEmissionsValue": null,
         "functionalUnitValue": null 
-      }
\ No newline at end of file
+      }
+    
+    marginal-carbon-intensity-forecast-example:
+      summary: An example response from an emissions forecast request.
+      value: [
+        {
+          "generatedAt": "2022-05-23T00:00:00Z",
+          "optimalDataPoint": {
+            "location": "eastus",
+            "time": "2022-05-23T00:10:00Z",
+            "rating": 173.3
+          },
+          "location": "eastus",
+          "notBefore": "2022-05-23T00:00:00Z",
+          "notAfter": "2022-05-23T00:21:00Z",
+          "forecastData": [
```

Comment:
Should we sort the forecast data by rating so that the consumer can pick the next best to optimal time? This is implementation detail, but may have to update the sample response if we decide to go that route.

Reply:
I think chronological order is a reasonable default given that we provide access to the best start time. This is a great future enhancement to the API to allow sorting/filtering though